PyRedPitaya/raw_memory.py

```diff
@@ -1,8 +1,22 @@
+import platform
 from ctypes import POINTER, c_uint32, cast, cdll, create_string_buffer, sizeof
+from os import path
 
 import numpy as np
 
-libmonitor_file = "libmonitor.so"
+libmonitor_file = path.join(
+    path.join(
+        path.dirname(__file__),
+        "..",
+        # linux-gnu is hardcoded since we are running this code on a red
+        # pitaya where it should be a GNU and not musl system, and most
+        # likely Linux
+        "monitor.cpython-{pyVer}-{machine}-linux-gnu.so".format(
+            pyVer="".join(list(platform.python_version_tuple())[0:2]),
+            machine=platform.machine(),
```

Comment:
This does not produce the correct name for me. The library is called `monitor.cpython-35m-arm-linux-gnueabihf.so` on the RedPitaya but this produces `monitor.cpython-35-armv7l-linux-gnu.so`.

Reply:
Thanks for testing. Now it should work, I used a better method then using `platform`.